REPORT_TEMPLATE.md

```diff
@@ -6,8 +6,6 @@ DO NOT modify it by hand as your changes will get lost the next time it is gener
 # Report template
 
 ```ruby
-# frozen_string_literal: true
-
```

Comment:
Can we leave this in whilst turning off the cop? We don't have it for "reasons" but agree with it in principle

Reply:
Went this way:
 - enforced to never add `frozen_string_literal`
 - excluded this doc file from the check